core/src/main/java/nl/nn/adapterframework/http/rest/ApiServiceDispatcher.java

```diff
@@ -298,9 +298,7 @@ private JsonObjectBuilder mapResponses(IAdapter adapter, MediaTypes contentType,
 			exit.add("description", status.getReasonPhrase());
 			if(!ple.getEmptyResult()) {
 				JsonObjectBuilder content = Json.createObjectBuilder();
-				if(schema == null) {
-					content.addNull(contentType.getContentType());
-				} else if(StringUtils.isNotEmpty(ref)){
+				if(StringUtils.isNotEmpty(ref)){
```

Comment:
Does this not break any tests? If not, we need to add them! It's important we also test what happens when there is no schema